daml-assistant/src/DA/Daml/Assistant/Env.hs

```diff
@@ -83,7 +83,7 @@ getLatestStableSdkVersion damlPath cachePath =
 testDamlEnv :: Env -> IO (Maybe Text)
 testDamlEnv Env{..} = firstJustM (\(test, msg) -> unlessMaybeM test (pure msg))
     [ ( doesDirectoryExist (unwrapDamlPath envDamlPath)
-      ,  "The daml home directory does not exist. Please check if DAML_HOME is incorrectly set, "
+      ,  "The Daml home directory does not exist. Please check if DAML_HOME is incorrectly set, "
```

Comment:
This makes sense to me, but probably worth having an extra pair of eyeballs from language confirming.

Reply:
Given that @remyhaemmerle-da already reviewed this and gave a green light and given that this is CLI feedback text, I believe it's safe to consider this item safe and resolve it.